src/Firestore/Connection/Grpc.php

```diff
@@ -80,7 +80,12 @@ public function __construct(array $config = [])
         $config['serializer'] = $this->serializer;
         $this->setRequestWrapper(new GrpcRequestWrapper($config));
 
-        $grpcConfig = $this->getGaxConfig(ManualFirestoreClient::VERSION);
+        $grpcConfig = $this->getGaxConfig(
+            ManualFirestoreClient::VERSION,
+            isset($config['authHttpHandler'])
```

Comment:
We have access to `ArrayTrait` via `GrpcTrait`. Could this use `ArrayTrait::pluck()` instead? ditto for other connection implementation classes.

Reply:
We don't need to pluck the item out of the array, though.